EarlGrey/Synchronization/GREYAppStateTracker.m

```diff
@@ -181,8 +181,8 @@ - (NSString *)grey_stringFromState:(GREYAppState)state {
     [eventStateString addObject:stateMsg];
   }
   if (state & kGREYPendingGestureRecognition) {
-    [eventStateString addObject:@"Waiting for system-wide gesture recognizer to detect or fail a "
```

Comment:
Removed because kGREYPendingGestureRecognition isn't only for system-wide gesture recognizers, it is for all of them.